src/esp/bindings/SimBindings.cpp

```diff
@@ -323,12 +323,27 @@ void initSimBindings(py::module& m) {
           "set_object_light_setup", &Simulator::setObjectLightSetup,
           "object_id"_a, "light_setup_key"_a, "scene_id"_a = 0,
           R"(Modify the LightSetup used to the render all components of an object by setting the LightSetup key referenced by all Drawables attached to the object's visual SceneNodes.)")
-
       .def(
           "get_num_active_contact_points",
           &Simulator::getNumActiveContactPoints,
-          R"(The number of contact points that were active during the last step. An object resting on another object will involve several active contact points. Once both objects are asleep, the contact points are inactive. This count can be used as a metric for the complexity/cost of collision-handling in the current scene.)");
-  ;
+          R"(The number of contact points that were active during the last step. An object resting on another object will involve several active contact points. Once both objects are asleep, the contact points are inactive. This count can be used as a metric for the complexity/cost of collision-handling in the current scene.)")
+      .def("get_vertices", &Simulator::getVertices, "ID"_a,
```

Comment:
```suggestion
      .def("get_vertices", &Simulator::getVertices, "id"_a,
```

Reply:
Lower case, or `mesh_key` or something if you are trying to avoid the builtin `id`